### ISSUE TITLE
Make EJ groups focused outreach plan and email first contact

### ISSUE THREAD

**Issue by saraannwylie** [edgi-govdata-archiving/EEW_Planning#182]
We decided our next priority for outreach is EJ groups, so want to create a specific plan for that and begin outreach.

┆Issue is synchronized with this [Trello card](https://trello.com/c/hb4waVcu)

**Comment by crgreenleaf** [comment]
Shall we chat tomorrow about this?

**Comment by lourdesvera** [comment]
@lesliealanis is taking lead on this issue. She will draw from the list of potential partners and draft an e-mail template to them for us to review. 

* [ ] spreadsheet of partner
* [ ] e-mail outreach template
* [ ] review
* [ ] send e-mail to one partner

**Comment by lourdesvera** [comment]
this is linked with #205

**Comment by lourdesvera** [comment]
➤ lourdesvera commented:

lesliealanis is taking lead on this issue. She will draw from the list of potential partners and draft an e-mail template to them for us to review. 

  * spreadsheet of partner
  * e-mail outreach template where we link to the landing page environmentalenforcementwatch.org and put in our e-mail address info@environmentalenforcementwatch.org
  * review
  * send e-mail to one partner

**Comment by lesliealanis** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Make EJ groups focused outreach plan and email first contact #182](https://trello.com/c/ACbRwpXJ/63-make-ej-groups-focused-outreach-plan-and-email-first-contact-182)

**Comment by lesliealanis** [comment]
Here is the email template, I feel like its too wordy so please feel free to look at it and make suggestions:
https://docs.google.com/document/d/1jzUdA8-6IhJY0BYdOn8ll4q9wiINJDQMJvsL86k0aN4/edit?usp=sharing

**Comment by lourdesvera** [comment]
We have a spreadsheet, template, and outreach to one EJ/DJ person (Monica)
